lib/ViewModels/AnimationViewModel.js

```diff
@@ -119,6 +124,74 @@ function updateForNewTopLayer(animationViewModel) {
     }
 }
 
+/**
+ * Set time to nearest time to specified (may be start or end of time range if time is not in range).
```

Comment:
This doc will show in in the generated jsdoc, which doesn't make sense because this function isn't accessible.  Either don't use jsdoc-style comments (regular comments are fine), or add `@private`.  Same with the one below.